modules/private.php

```diff
@@ -0,0 +1,307 @@
+<?php
+/**
+ * Module Name: Private site
+ * Module Description: Make your site only visible to you and users you approve.
+ * Sort Order: 9
+ * First Introduced: 7.2
```

Comment:
You'll need to update this reference (and the other ones) to `7.3`.

Reply:
Yup, pushed in the change as you were reviewing this :)
Done in [18ded36](https://github.com/Automattic/jetpack/pull/10333/commits/18ded36c5c009513b5c0e48b5463c45d389653b8)